DeviceTests/Caboodle.DeviceTests.Android/MainActivity.cs

```diff
@@ -18,7 +21,13 @@ protected override void OnCreate(Bundle bundle)
             var hostPort = Intent.Extras?.GetInt("HOST_PORT", 10578) ?? 10578;
 
             if (!string.IsNullOrEmpty(hostIp))
-                UnitTests.HeadlessRunner.Tests.RunAsync(hostIp, hostPort, typeof(Battery_Tests).Assembly);
+            {
+                var filters = new List<XUnitFilter>
+                {
+                    new XUnitFilter(Traits.DeviceType, Traits.DeviceTypes.ToExclude, true)
```

Comment:
We may need to address this in future if we ever have to add more filters - maybe find a way to sync them up or something. We just have to remember to do this for each platform project.

Reply:
Yeah I suppose we could have a shared helper method to `GetSharedFilters()`